### PR TITLE
feat(console): in-used connectors deletion alert

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): in-used connectors deletion alert

## Testing

Test locally.
image

### CODE REVIEW

File: packages/console/src/pages/ConnectorDetails/index.tsx [hunk 1]
```diff
@@ -49,6 +50,18 @@ const ConnectorDetails = () => {
   const api = useApi();
   const navigate = useNavigate();
   const theme = useTheme();
+  const isSocial = data?.type === ConnectorType.Social;
+  const [isDeleteAlertOpen, setIsDeleteAlertOpen] = useState(false);
+
+  const onDeleteClick = async () => {
+    if (!isSocial || !inUse) {
```
Comment (gao-sun) [inline packages/console/src/pages/ConnectorDetails/index.tsx:57]: non-social connectors should also have alert?

File: packages/console/src/pages/ConnectorDetails/index.tsx [hunk 2]
```diff
@@ -183,6 +194,22 @@ const ConnectorDetails = () => {
           />
         </Card>
       )}
+      {data && isSocial && inUse && (
```
Comment (gao-sun) [inline packages/console/src/pages/ConnectorDetails/index.tsx:197]: I don't think we need this check since the logic is independent

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/pages/ConnectorDetails/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/ConnectorDetails/index.tsx:57]: Per offline discussion, we only show this alert on the social connectors page.

File: packages/console/src/pages/ConnectorDetails/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/ConnectorDetails/index.tsx]: updated.

**Pull request merged**